src/js/pages/catalog/PackagesTab.js

```diff
@@ -225,31 +250,36 @@ class PackagesTab extends mixin(StoreMixin) {
       content = this.getLoadingScreen();
     } else {
       const packages = CosmosPackagesStore.getAvailablePackages();
-      const splitPackages = packages.getSelectedAndNonSelectedPackages();
 
-      let communityPackages = splitPackages.nonSelectedPackages;
-      const selectedPackages = splitPackages.selectedPackages;
-
-      if (state.searchString) {
-        communityPackages = packages.filterItemsByText(state.searchString);
-      }
-
-      content = (
-        <div className="container">
-          <div className="pod flush-horizontal flush-top">
-            <FieldAutofocus>
-              <FilterInputText
-                className="flex-grow"
-                placeholder="Search catalog"
-                searchString={state.searchString}
-                handleFilterChange={this.handleSearchStringChange}
-              />
-            </FieldAutofocus>
+      if (packages.list == null || packages.list.length === 0) {
```

Comment:
🤨- not sure if we are supposed to check on the `.list` property directly, I think it should be something like `packages.getItems().length`

Reply:
AFAIK someone is responsible to make all this a lot better right now #nopressure 🤪

Reply:
🎩 💥 🐇
Not better, insanely great! 

Although we all are responsible hehe, someone is just `facilitating` 😝